ext/phar/tests/024-opcache-win32.phpt

```diff
@@ -17,6 +17,7 @@ opcache.file_cache={PWD}/024-file_cache
 opcache.memory_consumption=64
 opcache.interned_strings_buffer=8
 opcache.max_accelerated_files=4000
+opcache.jit=tracing
```

Comment:
There are tons of tests that only specify `jit_buffer_size`. This no longer has any effect. IMO we should either:

1. Remove `jit_buffer_size ` altogether and rely on multiple test passes with/without JIT. This has the benefit of actually making sure the tests works consistently for vanilla, opcache, tracing and function JIT.
2. Add `opcache.jit=tracing` to all of them, to keep running tests with tracing JIT, as before.

@dstogov Are you ok with option 1? This does mean running JIT tests locally requires passing the `-dopcache.jit=tracing` option. I think that's acceptable.

Reply:
Option 1 looks right.

Reply:
@danog Can you please make the necessary changes?